spec/registry.md

```diff
@@ -1,20 +1,57 @@
 # MessageFormat 2.0 Default Function Registry
 
-This section describes the functions for which each implementation MUST provide
-a _function handler_ to be conformant with this specification.
-
-Implementations MAY implement additional _functions_ or additional _options_.
-In particular, implementations are encouraged to provide feedback on proposed
-_options_ and their values.
-
-> [!NOTE]
-> The [Stability Policy](/spec#stability-policy) allows for updates to
-> Default Registry functions to add support for new options.
-> As implementations are permitted to ignore options that they do not support,
-> it is possible to write messages using options not defined below
-> which currently format with no error, but which could produce errors
-> when formatted with a later edition of the Default Registry.
-> Therefore, using options not explicitly defined here is NOT RECOMMENDED.
+This section defines the **standard** _functions_ which are REQUIRED for conformance with this specification,
+along with **optional** _functions_ that SHOULD be implemented to support
+additional functionality.
+
+To **_<dfn>accept</dfn>_** a function means that an implementation MUST NOT
+emit an _Unknown Function_ error for that _function_'s _identifier_.
+To _accept_ an option means that a _function handler_ MUST NOT
+emit a _Bad Option_ error for that _option_'s _identifier_ when used with the _function_
+it is defined for
+and MUST NOT emit a _Bad Option_ error for any of the _option_ values
+(both **standard** and **optional**)
+defined for that _option_.
+Accepting a _function_ or its _options_ does not mean that a particular output is produced.
+Implementations MAY emit an _Unsupported Operation_ error for _options_
+or _option_ values that they cannot support.
+
+_Functions_ can define _options_. 
+An _option_ can be **standard** or **optional**.
+
+Implementations MUST _accept_ each **standard** _function_ and
+MUST _accept_ all _options_ and _option_ values defined as **standard** for those _functions_.
```

Comment:
```suggestion
MUST _accept_ all _options_ defined as **standard** for those _functions_.
```

Reply:
I'm nervous about this one, although I agree with your argument about standard/options for values. However, the requirement is meant to be that you cannot emit _Bad Option_ for a valid option value, e.g.:

```
.input {$n number maximumFractionDigits=2}
```

But you may do so for misspellings or otherwise bad values:

```
.input {$n :number maximumFActionDigit=2.5}
```

Reply:
We still allow emitting Bad Option for `2.5`, though? That's covered by the definition above:
> To _accept_ an _option_ means that a _function handler_ MUST NOT emit a _Bad Option_ error for that _option_'s _identifier_ when used with the _function_ it is defined for and MUST NOT emit a _Bad Option_ error for any of the _option_ values defined for that _option_.

Note that the final requirement only applies "for any of the _option_ values defined for that _option_", and `2.5` is not a valid _digit size option_ value.